app/Http/Requests/User/UserListingRequest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Http\Requests\User;
+
+use App\Http\Requests\AbstractEmptyRequest;
+use App\Models\User;
+use App\Policies\UserPolicy;
+use Illuminate\Support\Facades\Gate;
+
+class UserListingRequest extends AbstractEmptyRequest
```

Comment:
All other request classes start with the verb. For consistency this should be 
```suggestion
class ListUserRequest extends AbstractEmptyRequest
```

Reply:
fixed [87570d9](https://github.com/LycheeOrg/Lychee/pull/1539/commits/87570d9092a2f0bd7c55d605809d9f681de24a2f)